src/Fantomas/Context.fs

```diff
@@ -461,6 +461,7 @@ let internal sepNlnUnlessLastEventIsNewline (ctx: Context) =
 
 let internal sepStar = !- " * "
 let internal sepEq = !- " ="
+let internal sepEqFixed = !- "="
```

Comment:
Why the name "fixed" here?

Reply:
The suffix `fixed` is used to indicate that the character does not include any spaces regardless of settings.
[Other example](https://github.com/fsprojects/fantomas/blob/4d038e57127e4e1a19c595ef57ea6ab7ef7b677a/src/Fantomas/Context.fs#L479)